pkg/apis/eksctl.io/v1alpha5/vpc.go

```diff
@@ -21,6 +23,8 @@ type (
 		// for additional CIDR associations, e.g. to use with separate CIDR for
 		// private subnets or any ad-hoc subnets
 		// +optional
+		PodSubnets map[string]*CustomSubnets `json:"podSubnets,omitempty"`
```

Comment:
Do you think we could just add this as a new topology under `ClusterSubnets`?

Reply:
@errordeveloper As in `PodSubnets` under `ClusterSubnets`?